test/src/TapeTest.cpp

```diff
@@ -238,8 +238,8 @@ TEST_F(TapeTest, RecordPlayAllCalls)
 	rgl_node_t yield = nullptr;
 	EXPECT_RGL_SUCCESS(rgl_node_points_format(&yield, fields.data(), fields.size()));
 
-	rgl_node_t compact = nullptr;
-	EXPECT_RGL_SUCCESS(rgl_node_points_compact(&compact));
+	rgl_node_t compactByField = nullptr;
+	EXPECT_RGL_SUCCESS(rgl_node_points_compact_by_field(&compactByField, IS_HIT_I32));
```

Comment:
Until `rgl_node_points_compact` is completely deleted, it would be better to have both here.